proj/akka-persistence-jdbc.conf

```diff
@@ -1,9 +1,9 @@
-// https://github.com/dnvriend/akka-persistence-jdbc.git#master
+// https://github.com/dnvriend/akka-persistence-jdbc.git#v3.5.2
```

Comment:
@SethTisue Are you aware that it moved to akka/akka-persistence-jdbc and that's why we changed organisation id for the next major release?

Reply:
https://github.com/scala/community-build/pull/1016#issuecomment-564458333

Reply:
@ennru thanks! I'll switch to the new repo here once I see that downstream repos are expecting the new organization id